doc/README.tensorflow-horovod.md

```diff
@@ -0,0 +1,88 @@
+Instructions For Running TensorFlow Horovod
+-------------------------------------------
+
+0) Keep in mind that Magpie currently supports running TensorFlow Horovod
+   (TF) using OpenMPI on Slurm only. 
+   
+   _Disclaimer: Usually, directories mentioned below are placed on_
+   _a distributed file system (Lustre is recommended) directory,_
+   _so they are immediately and "by default" available from all nodes:_
+   
+   Download and install:
+   * TensorFlow
+   * OpenMPI
+   * Horovod
+   * EPEL
```

Comment:
I think you can remove EPEL, I think it's just assumed you are running on a system with an OS already :-)  I think OpenMPI doesn't have to be mentioned, as mpirun makes it obvious enough.

Reply:
OR ... is a specific OpenMPI version required?  Perhaps it should be listed if that's a requirement.

Reply:
You are absolutely right, these things sounds pretty obvious here, they were copied one-to-one from our internal documentation where we weren't using Magpie, and it was just note about our setup. Removing obvious things.